docs/advanced-features/event-driven-packaging.md

```diff
@@ -0,0 +1,177 @@
+<!-- markdownlint-disable -->
+# Event-Driven Packaging (EDP)
+
+> EDP is in private preview. Please contact us for details.
+
+## Overview
+
+Data tend to be created in logical batches by machines, people, and
+pipelines. Detecting these logical events from Amazon S3 events alone is
+complex and requires extensive custom logic.
+
+Quilt's *Event-Driven Packaging* service (EDP) smartly groups one
+or more Amazon S3 object events into a single event
+in [AWS EventBridge](https://aws.amazon.com/eventbridge/) so that
+you can easily trigger logical events like package creation that
+depend on batches rather than on individual files.
+
+Quilt EDP works well with services like [AWS Storage
+Gateway](https://aws.amazon.com/storagegateway/) and [AWS Data
+Sync](https://aws.amazon.com/datasync/) that write objects to Amazon S3.
+
+> Any AWS service or action that generates S3 object events may trigger the EDP service.
+
+## Requirements
+
+1. A pre-existing VPC that includes a [NAT
+Gateway](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway.html)
+or the following [VPC
+endpoints](https://docs.aws.amazon.com/vpc/latest/privatelink/concepts.html#concepts-vpc-endpoints):
+  - Amazon S3
+  ([gateway](https://docs.aws.amazon.com/vpc/latest/privatelink/gateway-endpoints.html)
+  or
+  [interface](https://docs.aws.amazon.com/vpc/latest/privatelink/create-interface-endpoint.html))
+  - EventBridge (interface)
+2. [Enable EventBridge S3
+Events](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-event-notifications-eventbridge.html)
+for all S3 buckets to be monitored by EDP
+
+![](../imgs/edp-event-bridge-s3-properties.png)
+
+## Deployment
+
+To use EDP in your Quilt instance, you will use a standalone
```

Comment:
EDP’s not tied to a Quilt instance, that’sa networking detail for elsewhere. Just say “is a standalone CloudFormation template”…